app/views/shared/_one_pager_simple.html.erb

```diff
@@ -0,0 +1,12 @@
+<div class="resource-link resource-doc">
+  <a
+    href="https://drive.google.com/open?id=1LzGcSK5nPS0izJT4kV7B3y6syl84ACqo&authuser=dburka%40resolvetosavelives.org&usp=drive_fs" 
```

Comment:
These static doc references would be worth pulling out to a constant or app helper...maybe something like `OnePageHelper` that has the links for various docs.  Its nice to keep static stuff like this consolidated in one file for when the URLs change in the future.